sys/shell/commands/sc_gnrc_netif.c

```diff
@@ -805,6 +987,20 @@ static int _netif_set(char *cmd_name, kernel_pid_t iface, char *key, char *value
     else if (strcmp("cca_threshold", key) == 0) {
         return _netif_set_u8(iface, NETOPT_CCA_THRESHOLD, 0, value);
     }
+    else if ((strcmp("frequency", key) == 0) || (strcmp("freq", key) == 0)) {
+        return _netif_set_u32(iface, NETOPT_CHANNEL_FREQUENCY, 0, value);
+    }
+#ifdef MODULE_LORA
```

Comment:
Dito. Usage above isn't `#ifdef`'d

Reply:
@miri64 I pushed 4ccfcfd that removes the ifdefs.